app/views/content/steps-to-become-a-teacher/_step_1_decide_who_to_teach.html.erb

```diff
@@ -0,0 +1,10 @@
+<p><a href="https://schoolexperience.education.gov.uk/">Get school experience</a> or <a href="/events">attend events</a> to see which of the following age groups suits you best:</p>
+
+<ul>
+  <li>primary (ages 5 to 11)</li>
+  <li>secondary (ages 11 to 18)</li>
+  <li>children up to the age of 5 - <a href="/early-years-teaching-training">become an early years teacher</a></li>
+  <li>people aged 14 and above - <a href="/further-education-teacher-training">become a further education teacher</a></li>
```

Comment:
Comes back to the perennial question of whether we can better define FE... I think >14 has the potential to be confusing. Wonder if we try and improve in this and the FE page.

> Further education (FE) includes any study after secondary education that’s not part of higher education (that is, not taken as part of an undergraduate or graduate degree).
> 
> Courses range from basic English and maths to Higher National Diplomas (HNDs).
> 
> FE also includes 3 types of technical and applied qualifications for 16 to 19-year-olds:
> 
> level 3 tech levels to specialise in a specific technical job
> level 2 technical certificates help get employment or progress to another tech level
> applied general qualifications to continue general education at advanced level through applied learning

Reply:
yea - definitely scope to improve that FE page I reckon. also just spotted there's some content on the beast page which we can weave in to the FE page: 

"Further education

To teach further education you do not need QTS.

Instead you can study for a PGCE or PGDE without QTS. These courses are also eligible for candidates without a degree.

You can search for further education courses leading to a post-compulsory education and training qualification (PCET). You do not need a degree to apply for courses leading to a PCET."